test/testgarbage.cpp

```diff
@@ -987,7 +987,7 @@ class TestGarbage : public TestFixture {
     }
 
     void garbageCode125() {
-        ASSERT_THROW_INTERNAL(checkCode("{ T struct B : T valueA_AA ; } T : [ T > ( ) { B } template < T > struct A < > : ] { ( ) { return valueA_AC struct { : } } b A < int > AC ( ) a_aa.M ; ( ) ( ) }"), UNKNOWN_MACRO);
```

Comment:
Some of these `UNKNOWN_MACRO` in the garbage tests felt undesired.

Reply:
We mostly use the "if it's uppercase, it's a macro" heuristic, so...